### PR TITLE
Master remove legacy fields aab

### PR DESCRIPTION
**Pull request by aab-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by aab-odoo** [comment]
robodoo rebase-ff

File: addons/web/static/tests/core/l10n/dates_tests.js [hunk 1]
```diff
@@ -13,1082 +13,709 @@ import {
...
+            ["2016-01-03 09:24:15.123+06:00", "2016-01-03T00:00:00.000Z"],
+            ["2016-01-03 09:24:15.123+16:00", "2016-01-02T00:00:00.000Z"],
+            ["2016-01-03 09:24:15.123Z", "2016-01-03T00:00:00.000Z"],
+        ]);
+
+        runTestSet(assert, testSet, (input) => parseDate(input).toISO());
+    });
+
+    QUnit.test("parseDateTime (various entries)", async (assert) => {
+        patchDate(2020, 6, 15, 12, 30, 0);
```
Comment (brboi) [inline addons/web/static/tests/core/l10n/dates_tests.js:585]: remove patchDate

File: addons/web/static/tests/core/l10n/dates_tests.js [hunk 2]
```diff
@@ -13,1082 +13,709 @@ import {
...
+        assert.strictEqual(
+            parseDateTime("20-10-20 8:5:3", { format: "yyyy-MM-dd hh:mm:ss" }).toISO(),
+            "2020-10-20T08:05:03.000+01:00"
         );
-    }
-);
+    });
+
+    QUnit.test("parseDate (various entries)", async (assert) => {
+        patchDate(2020, 6, 15, 12, 30, 0);
```
Comment (brboi) [inline addons/web/static/tests/core/l10n/dates_tests.js:477]: remove patchDate

**Review by brboi — COMMENTED** [review]
only reviewed dates_tests.js

**Comment by FrancoisGe** [comment]
robodoo r+

**Comment by mcm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**